migration-engine/connectors/sql-migration-connector/src/sql_renderer/mysql_renderer.rs

```diff
@@ -408,6 +410,7 @@ pub(crate) fn render_column_type(column: &ColumnWalker<'_>) -> Cow<'static, str>
             format!("ENUM({})", variants).into()
         }
         ColumnTypeFamily::Json => "json".into(),
+        ColumnTypeFamily::Binary => "mediumblob".into(),
```

Comment:
I think this prevents us from having a binary column as part of a primary key or unique constraint in general. I'll create an issue, let's tackle it later.

Reply:
Issue: https://github.com/prisma/prisma-engines/issues/1184

:heavy_check_mark: